peekingduck/loaders.py

```diff
@@ -84,6 +91,22 @@ def _load_node_list(self, run_config: str) -> List[str]:
         self.logger.info('Successfully loaded run_config file.')
         return nodes
 
+    def _get_custom_name_from_node_list(self) -> Any:
+
+        custom_name = None
+
+        for node in self.node_list:
+            if isinstance(node, dict) is True:
+                node_type = [*node][0].split(".")[0]
+            else:
+                node_type = node.split(".")[0]
+
+            if node_type not in PEEKINGDUCK_NODE_TYPE:
+                custom_name = node_type
+                break
+
+        return custom_name
```

Comment:
recommend taking custom_name from `cli` if possible.

Reply:
Understood that the cli is just for creating the folder not to control the input for `custom_folder_path`. Above comment is not relevant now.